runtime/vm/jvminit.c

```diff
@@ -2392,7 +2393,11 @@ static UDATA checkArgsConsumed(J9PortLibrary* portLibrary, J9VMInitArgs* j9vm_ar
 		if (IS_CONSUMABLE( j9vm_args, i ) && !IS_CONSUMED( j9vm_args, i )) {
 			char* optString = j9vm_args->actualVMArgs->options[i].optionString;
 			char* envVar = j9vm_args->j9Options[i].fromEnvVar;
-
+			
+			/* If an option is empty or contains only whitespace, it should be ignored instead of rejected */
+			if (optString && isEmpty(optString)) {
```

Comment:
Please be explicit with the NULL check: `(NULL != optString) && isEmpty(optString)`